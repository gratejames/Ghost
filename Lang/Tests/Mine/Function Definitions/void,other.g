int main(void, int x) {
    // test case w/ multi-digit constant
    return 100;
}