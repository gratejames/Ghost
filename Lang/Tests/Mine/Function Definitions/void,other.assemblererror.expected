If 'void' is the argument, it must stand alone.
SAD (No AST output...)
